services/aca-py/Dockerfile.acapy

```diff
@@ -0,0 +1,10 @@
+FROM bcgovimages/aries-cloudagent:py36-1.16-1_0.7.4-rc0
```

Comment:
this is only for local development and running - correct? not to be deployed on openshift?

Reply:
Ah right ...  I updated to 0.7.4-rc0 and there have been no issues, however we may want to wait for the "official" 0.7.4 release before we update the aca-py version on openshift.

FYI I expect `0.7.4-rc2` (I think is what we are up to) to come out this week, and then an official `0.7.4` next week.

With the official `0.7.4` we'll need to re-visit the aca-py client and wrapper libraries (openapi-generated code).  There are no differences in the new aca-py version for the api's that we're using, but there are changes to other api's and we should make sure we match ...